Fitness/Extensions/String+Shared.swift

```diff
@@ -6,10 +6,22 @@
 //  Copyright © 2018 Uplift. All rights reserved.
 //
 
-import Foundation
+import UIKit
```

Comment:
j curious, what's the dif?

Reply:
UIKit is what gives you all the UI components and CGFloat